[DOCS-7775] Add troubleshooting topic in Search and Insight Engine 2.0 (#1600)
Alfresco/docs-alfresco: insight-engine/latest/admin/monitor.md

The task assumes you are using only one Solr instance for all nodes in the Alfresco Content Services cluster. If not, then you need to repeat the process on each Solr instance used in the cluster.

> 1. Confirm the location of the Solr core directories for archive and alfresco cores. This can be determined from the solrcore.properties file for both the cores. By default, the `solrcore.properties file` can be found at `<SOLR_HOME>/solrhome/alfresco/conf` or `<SOLR_HOME>/solrhome/archive/conf`. The Solr core location is defined in the `solrcore.properties` file as:

Reviewer: ```suggestion
1. Confirm the location of the Solr core directories for archive and alfresco cores. This can be determined from the solrcore.properties file for both the cores. By default, the `solrcore.properties` file can be found at `<SOLR_HOME>/solrhome/alfresco/conf` or `<SOLR_HOME>/solrhome/archive/conf`. The Solr core location is defined in the `solrcore.properties` file as:
```

You can also fix index issues, check the index cache and backup individual indexes by using JMX. The status of the index can be checked using the JMX client on the **JMX MBeans > Alfresco > solrIndexes > `<store>`** tabs. The default view is the Solr core summary. The operations run the same consistency checks that are available by URL.

> ### Troubleshooting - unsearchable content

Author: I thought to add it as an extra heading because the formatting in the Troubleshooting topic is very different. In the Troubleshooting you only have one-liners, and this is a longer procedure. Also, it is specified at the beginning of that topic that there is information on how to repair a transaction that failed to index only. Maybe adding "transaction failed to index" to the "Troubleshooting" section name and changing the heading level of the added section to two would be better? So that it's not a subtopic of the original Troubleshooting.

> ### Troubleshooting - unsearchable content

Reviewer: I like your suggestion 👍🏼

If content becomes unsearchable for some documents, it is possible that the necessary transformations failed and the documents are currently indexed with a `transform_failed` status.

> > **Note:** For Enterprise installations (assuming the reason for these failed transformations is the temporary unavailability of the Transform Engines (T-Engines)) it is recommended to verify that the T-Engines do get back up and running, and to check that ActiveMQ was and still is available. In that case, the documents indexed with a `transform_failed` status should be automatically reindexed once the T-Engines come back up and the prior ActiveMQ messages are consumed, without any further steps being required.

Reviewer: Hi @damianujma - I've changed the wording here - is it still technically correct? 
```suggestion
> **Note:** For Enterprise installations (assuming the reason for these failed transformations is the temporary unavailability of the Transform Engines (T-Engines)), it is recommended to verify that the T-Engines start running again, and check that ActiveMQ is still available. In that case, the documents indexed with a `transform_failed` status should be automatically reindexed once the T-Engines start running again and the earlier ActiveMQ messages are consumed, without any further steps being required.
```

Reviewer 2: LGTM.
